src/Sylius/Bundle/AdminBundle/Controller/CustomerStatisticsController.php

```diff
@@ -29,13 +29,13 @@ final class CustomerStatisticsController
     /** @var RepositoryInterface */
     private $customerRepository;
 
-    /** @var EngineInterface */
+    /** @var Environment */
     private $templatingEngine;
 
     public function __construct(
         CustomerStatisticsProviderInterface $statisticsProvider,
         RepositoryInterface $customerRepository,
-        EngineInterface $templatingEngine
+        Environment $templatingEngine
```

Comment:
Is it backward compatible change?

Reply:
Hard to say, from what I know they dropped the support for templating et all.